chrysom/client.go

```diff
@@ -386,7 +392,7 @@ func (c *Client) Start(ctx context.Context) error {
 				return
 			case <-c.observer.ticker.C:
 				outcome := SuccessOutcome
-				items, err := c.GetItems("")
+				items, err := c.GetItems(context.Background(), "")
```

Comment:
Unfortunately, this context won't have a logger for `GetItems()` to extract. 😞 If we add `c.logger` to this context before sending it, I think that will fix this. 👍

Reply:
Thanks for giving the feedback. But this is already handled. C.getLogger(ctx) will be calling Tr1d1um Getlogger(ctx) which internally calls GetLogger from Webpa-common/logging and there it is handled.

**References** 
Tr1d1um: https://github.com/xmidt-org/tr1d1um/blob/a859a3f0c97915f193b6d6aa37de1e52d5ce7ad8/main.go#L356-L359
Webpa-common/logging: https://github.com/xmidt-org/webpa-common/blob/main/logging/context.go#L21-L25

Reply:
I don't think we want to intentionally fallback to the default logger from the `webpa-common/logging` package:
https://github.com/xmidt-org/webpa-common/blob/90256afe7c190705e30ed769814aa56475580f4b/logging/logger.go#L12

Reply:
In applications like tr1d1um and scytale, the logger is configured through the configuration file.  I think it would make for more consistent logging if the argus client uses the logger passed by the application (with the proper configuration) in the long running goroutine.

Reply:
If this is the case, I will set c.logger in the context but it won't have the fields which are set [here](https://github.com/xmidt-org/tr1d1um/blob/a859a3f0c97915f193b6d6aa37de1e52d5ce7ad8/main.go#L350).

Reply:
That's okay, because this request to argus isn't a part of a request that tr1d1um or scytale received; it's originating in this goroutine.  Even for tracing, the tracing information would start with this client, right?

Reply:
It won't start here. It will be getting started in TraceMiddleware of Argus, here in the response we will be getting the X-Midt-Trace-ID and X-Midt-Span-ID